src/datasources/balances-api/zerion-balances-api.service.ts

```diff
@@ -24,23 +25,32 @@ import {
 import { Collectible } from '@/domain/collectibles/entities/collectible.entity';
 import { getNumberString } from '@/domain/common/utils/utils';
 import { Page } from '@/domain/entities/page.entity';
+import { DataSourceError } from '@/domain/errors/data-source.error';
 import { IBalancesApi } from '@/domain/interfaces/balances-api.interface';
+import { LoggingService, ILoggingService } from '@/logging/logging.interface';
 import { Inject, Injectable } from '@nestjs/common';
 
 export const IZerionBalancesApi = Symbol('IZerionBalancesApi');
 
 @Injectable()
 export class ZerionBalancesApi implements IBalancesApi {
   private static readonly collectiblesSorting = '-floor_price';
+  private static readonly RATE_LIMIT_CACHE_KEY_PREFIX = 'zerion';
```

Comment:
Nit: I'd either match the casing of `collectiblesSorting` or change that to screaming snake case. I'm in favour of the latter.